tests/unit/scripting.tcl

```diff
@@ -616,12 +616,12 @@ start_server {tags {"scripting"}} {
         set rd [redis_deferring_client]
         r config set lua-time-limit 10
         $rd eval {local f = function() while 1 do redis.call('ping') end end while 1 do pcall(f) end} 0
-        after 200
-        catch {r ping} e
-        assert_match {BUSY*} $e
```

Comment:
i think we want to wait to see the BUSY message, otherwise the script KILL may fail if the script (rd) didn't start yet.

p.s. let's also add `$rd close` at the bottom